scripts/devnet/.env

```diff
@@ -20,4 +20,4 @@ DRAGON_HEIGHT=12
 WAFFLE_HEIGHT=18
 TUKTUK_HEIGHT=20
 TEEP_HEIGHT=22
```

Comment:
```suggestion
TEEP_HEIGHT=-1
```

Reply:
@hanabi1224 setting `TEEP_HEIGHT=-1` doesnt seem to work

Reply:
Perhaps set it to 99999999